data/lfaidata.yaml

```diff
@@ -304,23 +304,6 @@
       check_sets:
         - community
         - code
-- name: kserve
```

Comment:
Is the [KServe project](https://landscape.lfai.foundation/?item=model--inference--kserve) no longer part of LF AI & Data?